frontend/tailwind.config.js

```diff
@@ -1,9 +1,12 @@
 const colors = require('tailwindcss/colors')
 
 module.exports = {
+  content: [
+    "./index.html",
+    "./src/**/*.{vue,js,ts,jsx,tsx}",
+  ],
   mode: 'jit',
-  purge: { content: ['./public/**/*.html', './src/**/*.vue'] },
-  darkMode: false, // 'class'
+  darkMode: 'class',
```

Comment:
Dit is nog niet op elke pagina af, dus kan niet echt kwaad, want er is nog geen optie voor gebruikers om naar darkmode te switchen, maar dat was de reden dat ik het voor de zekerheid nog even op `false` had staan.